src/components/WelcomeForm/Message.jsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import styles from "./Message.module.css";
+
+const Message = ({ user }) => {
+  if (!user.userName && !user.age) {
+    return null;
+  }
+
+  if (isNaN(user.age) || user.age == "") {
```

Comment:
Here since you converted the `age` to a number you only have two options or a number or is null, you can refactor this `if(user.age) {...` this is going to be true if the value is not null, and the numeric value is greater than 0, I encourage you to learn more about Truthy and Falsy values in Javascript 
https://developer.mozilla.org/en-US/docs/Glossary/Truthy
https://developer.mozilla.org/en-US/docs/Glossary/Falsy